cuml/README.md

```diff
@@ -0,0 +1,104 @@
+# cuML Notebooks
+
+## Summary
+
+* `dbscan_demo`: Notebook showcasing density-based spatial clustering of applications with noise (dbscan) algorithm comparison between cuML and scikit-learn.
+* `knn_demo`: Notebook showcasing k-nearest neighbors (knn) algorithm comparison between cuML and scikit-learn.
+* `pca_demo`: Notebook showcasing principal component analysis (PCA) algorithm comparison between cuML and scikit-learn.
+* `dask-setup.sh`: Notebook showcasing truncated singular value decomposition (tsvd) algorithm comparison between cuML and scikit-learn.
+
+## dbscan_demo
+
+Typical output of the cells processing dbscan looks like:
+
+- For scikit-learn dbscan:
+```
+CPU times: user 25.9 s, sys: 315 ms, total: 26.2 s
+Wall time: 26.2 s
+```
+
+- For cuML dbscan:
+```
+CPU times: user 6.02 s, sys: 19.7 ms, total: 6.04 s
+Wall time: 6.11 s
+```
+
+Final cell of the notebook should output:
+
+```
+compare dbscan: cuml vs sklearn labels_ equal
+```
+
+Note that the timing differences depend upon the exact dataset being used. Also, the dbscan algorithm implemented in version 0.4.0 of cuML is equivalent to `algorithm=brute` in [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.cluster.DBSCAN.html)
+
+## knn_demo
+
+The knn_demo notebook showcases the connection that cuML provides between `cuDF` and `faiss-gpu`. Currently, it is supported to run without any additional steps for CUDA 9.2, but it requires building `faiss-gpu` from source for CUDA 10.0 with [GPU support](https://github.com/facebookresearch/faiss/blob/master/INSTALL.md).
```

Comment:
>The knn_demo notebook showcases the connection that cuML provides between `cuDF` and `faiss-gpu`. Currently, it is supported to run without any additional steps for CUDA 9.2, but it requires building `faiss-gpu` from source for CUDA 10.0 with [GPU support](https://github.com/facebookresearch/faiss/blob/master/INSTALL.md).

Should read:

The `knn_demo` notebook demonstrates how `cuml` establishes interoperability between `cudf` and `faiss-gpu`. There is native support for this demo with CUDA 9.2. With CUDA 10.0, the user must build `faiss-gpu` from source with [GPU support](https://github.com/facebookresearch/faiss/blob/master/INSTALL.md)